### ISSUE TITLE
HID remapper doesn't monitor any game controller except mouse  and keyboard

### ISSUE THREAD

**Issue by pftmclub** [jfedor2/hid-remapper#193]
Hello, i'm trying to remap my GP2040 controller, but none of them working . Tried it with keyboard and it's work successfully, does  anyone know this  problem?

Gamepad:

https://github.com/user-attachments/assets/c69d0429-65f2-4e99-80c4-f71d3a473d2d

Keyboard:

https://github.com/user-attachments/assets/e6717dcd-671b-4fcc-85de-6fc36c832ff7

**Comment by jfedor2** [comment]
I just tested that it works in Switch mode for example.

HID Remapper doesn't care about the shape of the plug as long as the data lines are wired properly. Does your keyboard work with the same cable?

**Comment by pftmclub** [comment]
> I just tested that it works in Switch mode for example.
> 
> HID Remapper doesn't care about the shape of the plug as long as the data lines are wired properly. Does your keyboard work with the same cable?
My  keyboard have usb cable built-in ,

I tried to plug the USB-C directly into the usb hud and it's flashing. The other case is plug the Printer cable (USB-B) , nothing seem work .

USB-C:
![IMG_20241130_222626](https://github.com/user-attachments/assets/d28a5bd1-ab54-4a6c-b749-d40d2b10ccb2)

USB-B :
![IMG_20241130_222731](https://github.com/user-attachments/assets/92c1e9c9-d160-491e-8ffa-623b87428dce)

**Comment by jfedor2** [comment]
So it works when you connect it directly, but doesn't work when it goes through the Neutrik-or-whatever-style port on the enclosure? I don't see why it wouldn't.

**Comment by pftmclub** [comment]
> So it works when you connect it directly, but doesn't work when it goes through the Neutrik-or-whatever-style port on the enclosure? I don't see why it wouldn't.

Yeah, and its strange that the controller still function over OLED screen but the remapper doesn't recognize it.
To put it simple , i plug the cable to usb-c to usb-b hud , then using the usb-b cable into usb hud of the pico board
